Include/pyhash.h

```diff
@@ -12,8 +12,23 @@ PyAPI_FUNC(Py_hash_t) _Py_HashPointer(void*);
 PyAPI_FUNC(Py_hash_t) _Py_HashBytes(const void*, Py_ssize_t);
 #endif
 
-/* Prime multiplier used in string and various other hashes. */
-#define _PyHASH_MULTIPLIER 1000003UL  /* 0xf4243 */
+/* Multiplier used for various hashes */
+#define _PyHASH_MULTIPLIER 1000003UL
+
+/* Official constants from the xxHash specification */
```

Comment:
Suggest adding:

    /* Optimizing compilers should emit a single "rotate" instruction for the _PyHASH_XXROTATE
     * expansion.   If that doesn't happen for some important platform, the macro could be changed
     * to expand to a platform-specific rotate spelling instead.
     */